diff --git a/autoload/Colorizer.vim b/autoload/Colorizer.vim
index 9ade4fd..0f77a75 100644
--- a/autoload/Colorizer.vim
+++ b/autoload/Colorizer.vim
@@ -1537,7 +1537,7 @@ function! s:ColorInit(...) "{{{1
         \ 'term': [ '\%(\%(\%x1b\|\\033\)\[0m\)\?\(\%(\%(\%x1b\|\\033\)\[\d\+\%([:;]\d\+\)*m\)\+\)\([^\e]*\)\(\%(\%x1b\|\\033\)\%(\[0m\|\[K\)\)\=',
             \ function("s:PreviewColorTerm"), 'colorizer_term', [] ],
         \ 'term_nroff': ['\%(\(.\)\%u8\1\)\|\%(_\%u8.\)', function("s:PreviewColorNroff"), 'colorizer_nroff', [] ],
-        \ 'term_conceal': [ ['\%(\(\%(\%x1b\[0m\)\?\%x1b\[[02-9]\+\%([;:]\d\+\)*\a\)\|\%x1b\[K$\)',
+        \ 'term_conceal': [ ['\%(\(\%(\%x1b\[0m\)\?\%x1b\[[0-9]\+\%([;:]\d\+\)*\a\)\|\%x1b\[K$\)',
           \ '\%d13', '\%(\%x1b\[K\)', '\%(\%x1b\]\d\+;\d\+;\)', '\%(\%x1b\\\)',
           \ '\%x1b(B\%x1b\[m', '\%x1b\[m\%(\%x0f\)\?', '_\%u8.\@=', '\(.\)\%u8\%(\1\)\@=', '\%x1b\[[0-1]m'], 
           \ '',
